source/libs/page-detect.ts

```diff
@@ -60,7 +60,7 @@ export const isPR = (): boolean => /^pull\/\d+/.test(getRepoPath()!);
 
 export const isConflict = (): boolean => /^pull\/\d+\/conflicts/.test(getRepoPath()!);
 
-export const isPRList = (): boolean => getRepoPath() === 'pulls';
+export const isPRList = (): boolean => location.pathname === '/pulls' || getRepoPath() === 'pulls';
```

Comment:
This could potentially break features that rely on `isPRList` and assume it's a repo PR list.

Reply:
This is fine. All PR-list related features should work on all PR lists.